### ISSUE TITLE
Returned token is invalid

### ISSUE THREAD

**Issue by AndreasHald** [fechanique/cordova-plugin-fcm#492]
Hi, we have been using the plugin awhile to send notifications to our users devices, however suddenly firebase messaging rejects the tokens returned by fcm.getToken();

firebase messaging provides the error code 'messaging/invalid-argument'

![screen shot 2018-04-06 at 11 26 43](https://user-images.githubusercontent.com/8914722/38414264-4c96c500-398e-11e8-8bfd-25e902ff8ef6.png)

I can't even send a notification to the token directly from the firestore console

![screen shot 2018-04-06 at 11 27 48](https://user-images.githubusercontent.com/8914722/38414298-6864d7b8-398e-11e8-9482-1659b3934b86.png)
 

Anyone having similar issues?

**Comment by EliakimRamos** [comment]
I also have but I can not solve you solved?

**Comment by AndreasHald** [comment]
No, we couldn't fix it and moved to a different notifications provider OneSignal.
